Week3/homework/step2-7.js

```diff
@@ -21,3 +21,9 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+/*
+---pass by value 
+the parameter of "val" has nothing to do with argument "x". Const x is beyond the function scope.
```

Comment:
While this is true, it's only part of the story.
It's also important to understand the difference between changing the value of a variable (like changing x from 9 to 10), and changing the a property of a variable (like changing val.x from 9 to 10). Have a read through some of the discussions on this page https://stackoverflow.com/questions/6605640/javascript-by-reference-vs-by-value